RECORDINGS.md

```diff
@@ -0,0 +1,53 @@
+# Recordings
+
+This file describes the steps needed to set up an environment to record images and videos for Plone 6 Documentation.
+
+
+## Set up the environment
+
+Open a terminal session and issue the following command.
+
+```shell
+make recording-init
+```
+
+The command will run Cookieplone, generator a project, and install it.
+
+This project will serve as baseline for the tests. Eventually, we could pull a specific branch for generating the tests.
+
+Now you will start the backend, frontend, and acceptance test servers, one each in its own terminal session.
+
+```{note}
+None of these commands are documented anywhere.
+You could run `make help`, and get a dump of the commands, but that lacks context for usage.
+See https://github.com/plone/documentation/issues/1758.
+```
```

Comment:
@stevepiercy this is not true, the commands do what you say below, and in the Makefile help.